packages/react-themeless-provider/package.json

```diff
@@ -6,12 +6,18 @@
   },
   "main": "./src/index.js",
   "typings": "./src/index.d.ts",
-  "private": true,
+  "devDependencies": {
+    "@fluentui/react-jsx-runtime": "^9.0.32",
+    "@fluentui-contrib/react-shadow": "^0.1.0"
+  },
   "peerDependencies": {
     "@fluentui/react-components": ">=9.46.3 <10.0.0",
+    "@fluentui/react-icons": ">=2.0.224 <3.0.0",
+    "@fluentui/react-jsx-runtime": ">=9.0.32 <10.0.0",
+    "@fluentui/react-shared-contexts": ">=9.7.2 <10.0.0",
+    "@fluentui/react-theme": ">=9.1.16 <10.0.0",
+    "@griffel/react": ">=1.5.14 <2.0.0",
```

Comment:
We should probably also put `@griffel/react` as a regular dep.

I don't think that it's a good idea to have anything else except `@fluentui/react-components` as `peerDependency` as it's a single package that we expect to be installed by consumers.